app/src/adapters/db/postgres_client.py

```diff
@@ -0,0 +1,146 @@
+import logging
```

Comment:
when you're cleaning up some comments would you mind adding one sentence blurb explaining what this file is i.e. it provides the a db client that interacts with a postgres database.